content/courses/onchain-development/anchor-cpi.md

```diff
@@ -212,8 +211,8 @@ possible options:
    similar pattern if you decide to use an accounts struct and `CpiContext` to
    organize and prepare your CPI.
    ```rust
-   pub fn mint_to<'a, 'b, 'c, 'info>(
-       ctx: CpiContext<'a, 'b, 'c, 'info, MintTo<'info>>,
+   pub fn mint_to<'info>(
+       ctx: CpiContext<'_, '_, '_, 'info, MintTo<'info>>,
```

Comment:
I know these are unused, but can we name them anyway?

> '_foo, '_bar, '_baz'

But using whatever names CpiContext uses?

Reply:
sure ig, sounds fine